src/main/java/racingcar/model/Car.java

```diff
@@ -0,0 +1,32 @@
+package racingcar.model;
+
+import static racingcar.util.validator.CarNameValidator.checkOverRange;
+import static racingcar.util.validator.CarNameValidator.checkSpace;
+
+public class Car {
+
+    private int position;
+    private final String name;
+    private static final int MIN_FORWARD_REQUIREMENT = 4;
+
+    public Car(String name) {
+        checkOverRange(name);
+        checkSpace(name);
+
+        this.name = name;
+    }
```

Comment:
보통 이와 같은 매직넘버는 

```suggestion
public class Car {

    private static final int MIN_FORWARD_REQUIREMENT = 4;

    private int position;
    private final String name;

    public Car(String name) {
        checkOverRange(name);
        checkSpace(name);

        this.name = name;
    }
```

상단에 두곤합니다.